fwoa-tools/src/migrationUtils.test.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { createWriteStream, WriteStream } from 'fs';
+import AWS from 'aws-sdk';
+import { InitiateAuthRequest } from 'aws-sdk/clients/cognitoidentityserviceprovider';
+import * as AWSMock from 'aws-sdk-mock';
+import { checkConfiguration, getFhirClient, getFhirClientSMART } from './migrationUtils';
+
+const env = process.env;
```

Comment:
nit: can we also add an import of `dotenv` and add importing the environment from the `./.env` file (similar to how its done in the scripts). This could make it easier to test by having a separate ./env file (maybe named unit_test.env or something) with all the unit test credentials.